scripts/Testing.psm1

```diff
@@ -0,0 +1,144 @@
+function New-TestCommand {
+    [CmdletBinding()]
+    param (
+        [ValidateSet('Test', 'Bench', 'List')]
+        [alias('a')]
+        [string]
+        $Action = 'Bench',
+
+        [Parameter(Mandatory)]
+        [string]
+        $Path,
+
+        [Parameter(Mandatory)]
+        [string]
+        $Name,
+
+        [Parameter(Mandatory)]
+        [string]
+        $OutDirectory ,
+
+        [DateTime]
+        $Date = (Get-Date),
+
+        [string]
+        $Note = '',
+
+        # test parameters
+        [alias('tv')]
```

Comment:
Nit: I feel like -v is a pretty universal flag alias for verbose. Is there a reason you used -tv instead?